modules/oe_theme_list_pages/oe_theme_list_pages.info.yml

```diff
@@ -0,0 +1,9 @@
+name: OpenEuropa Theme List Pages
```

Comment:
Usually the name of the module is "oe_theme_content_list_pages".

Reply:
If it's from oe_content...this is not from oe_content..